src/components/SEBankIDSameDeviceButton.tsx

```diff
@@ -89,11 +95,11 @@ export default function SEBankIDSameDeviceButton(props: Props) {
     .then(links => {
       setPKCE(pkce || undefined);
       setLinks(links);
-      const redirect = mobileOS === 'iOS' ? encodeURIComponent(window.location.href) : 'null';
-      const useUniveralLink =
-        mobileOS === 'iOS' ? userAgent?.browser.name?.includes('Safari') :
-        mobileOS === 'android' ? userAgent?.browser.name === 'Chrome' :
-        false;
+
+      const iOSSafari = mobileOS === 'iOS' && userAgent?.browser.name?.includes('Safari') ? true : false;
```

Comment:
Looks like a duplicate of the same variable name defined at the top of the exported fn ?